src/api/app/controllers/webui/request_controller.rb

```diff
@@ -497,13 +497,21 @@ def staging_status(request, target_project)
     return nil unless (staging_review = request.reviews.staging(target_project).last)
 
     if staging_review.for_project?
-      { name: staging_review.project.name[target_project.name.length + 1..],
-        title: "staged in #{staging_review.project}",
-        url: staging_workflow_staging_project_path(target_project.name, staging_review.project.name) }
-    elsif staging_review.for_group?
-      { name: staging_review.accepted? ? 'Staged' : 'Unstaged',
-        title: 'submitted against a staging project',
-        url: staging_workflow_path(@bs_request.target_project_name) }
+      staging_project = {
+        name: staging_review.project,
```

Comment:
People refer to this as `Staging:Something`. 

Some recent example from the [opensuse-factory](https://lists.opensuse.org/archives/list/factory@lists.opensuse.org/thread/X44WML3GTW3S7ZTZDZMEU4EUPOQVWDJU/) mailing list:

>  Staging:O: preparation work to identify the minimal list of 'i586' builds to enable in openSUSE:Factory to allow wine/steam to build/install

That's the reason you had this abbreviation in the code you removed

```
irb(main):003:0> target_project_name = 'openSUSE:Factory'
=> "openSUSE:Factory"
irb(main):004:0> 'openSUSE:Factory:Staging:O'[target_project_name.length + 1..]
=> "Staging:O"
``